settings.py

```diff
@@ -551,6 +555,7 @@ def JINJA_CONFIG():
 # in DOMAIN_METHODS.
 # http://github.com/jbalogh/zamboni/blob/d4c64239c24aa2f1e91276909823d1d1b290f0ee/settings.py#L254
 STANDALONE_DOMAINS = ['django', 'javascript']
+STATICI18N_ROOT = 'build/locale'
```

Comment:
It defaults to `STATIC_ROOT` which is the `kuma/` folder. We want to write it to `build/locale` where pipeline will pick it up and put it in `static/`. I tested and this works in my VM. If you're fine with it I'll leave it as-is.